src/mutations/item.ts

```diff
@@ -337,9 +337,17 @@ export default (queryConfig: QueryClientConfig) => {
 
   const useCopyItems = () =>
     useMutation(
-      ({ ids, to }: { ids: UUID[]; to?: UUID }) =>
+      ({
+        ids,
+        to,
+        newNames,
+      }: {
+        ids: UUID[];
+        to?: UUID;
+        newNames?: { [key: string]: string };
+      }) =>
         splitRequestByIds(ids, MAX_TARGETS_FOR_MODIFY_REQUEST, (chunk) =>
-          Api.copyItems({ ids: chunk, to }, queryConfig),
+          Api.copyItems({ ids: chunk, to, newNames }, queryConfig),
```

Comment:
Since this call is split to accommodate for the maximum number of target items, we should maybe do the same splitting on the newNames to be consistent, even if in this case sending more data that is not useful to the call won't do any harm, I think it will be more future-proof